http-service/src/main/java/net/runelite/http/service/pluginhub/PluginHubController.java

```diff
@@ -86,6 +89,54 @@ public ResponseEntity<Map<String, Long>> get()
 			.body(pluginCounts);
 	}
 
+	@GetMapping("/shields/installs/plugin/{pluginName}")
+	public ResponseEntity<ShieldsFormat> installs(@PathVariable String pluginName)
+	{
+		if (pluginCounts.isEmpty())
+		{
+			return ResponseEntity.status(HttpStatus.SERVICE_UNAVAILABLE)
+				.cacheControl(CacheControl.noCache())
+				.build();
+		}
+
+		long count = pluginCounts.getOrDefault(pluginName, -1L);
+		return ResponseEntity.ok()
+			.cacheControl(CacheControl.maxAge(30, TimeUnit.MINUTES).cachePublic())
+			.body(new ShieldsFormat(count, "Total installs"));
+	}
+
+	@GetMapping("/shields/rank/plugin/{pluginName}")
+	public ResponseEntity<ShieldsFormat> rank(@PathVariable String pluginName)
+	{
+		if (pluginCounts.isEmpty())
+		{
+			return ResponseEntity.status(HttpStatus.SERVICE_UNAVAILABLE)
+				.cacheControl(CacheControl.noCache())
+				.build();
+		}
+
+		// check if the plugin exists before sorting by installation counts for minor performance optimization
+		long rank = pluginCounts.getOrDefault(pluginName, -1L);
+		if (rank != -1)
+		{
+			List<Map.Entry<String, Long>> orderedEntries = pluginCounts.entrySet().stream()
```

Comment:
Instead of sorting + counting to get the rank, you can just count the number of plugins with more installs

Reply:
Implemented your change, now plugins with the same install count will share a rank.
![image](https://github.com/runelite/api.runelite.net/assets/29030969/8456320e-be70-4d3d-85fe-57f82445e950)
![image](https://github.com/runelite/api.runelite.net/assets/29030969/cf8f206b-d6c9-4e65-b7f8-49b81469868a)